bundles/org.openhab.binding.http/README.md

```diff
@@ -0,0 +1,118 @@
+| `closedValue`           | no      |      -      | A special value that represents `CLOSED` |
+
+### `dimmer`
+
+| parameter               | optional | default     | description |
+|-------------------------|----------|-------------|-------------|
+| `onValue`               | yes      |      -      | A special value that represents `ON` |
+| `offValue`              | yes      |      -      | A special value that represents `OFF` |
+| `increaseValue`         | yes      |      -      | A special value that represents `INCREASE` |
+| `decreaseValue`         | yes      |      -      | A special value that represents `DECREASE` |
+| `step`                  |          |      1      | The amount the brightness is increased/decreased on `INCREASE`/`DECREASE` |
+
+All values that are not `onValue`, `offValue`, `increaseValue`, `decreaseValue` are interpreted as brightness 0-100% and need to be numeric only.
+
+### `rollershutter`
+
+| parameter               | optional | default     | description |
+|-------------------------|----------|-------------|-------------|
+| `upValue`               | yes      |      -      | A special value that represents `UP` |
+| `downValue`             | yes      |      -      | A special value that represents `DOWN` |
+| `stopValue`             | yes      |      -      | A special value that represents `STOP` |
+| `moveValue`             | yes      |      -      | A special value that represents `MOVE` |
+
+All values that are not `upValue`, `downValue`, `stopValue`, `moveValue` are interpreted as position 0-100% and need to be numeric only.
+                    
+### `switch`
+
+| parameter               | optional | default     | description |
+|-------------------------|----------|-------------|-------------|
+| `onValue`               | no       |      -      | A special value that represents `ON` |
+| `offValue`              | no      |      -      | A special value that represents `OFF` |
+
+## URL Formatting
+
+After concatenation of the `baseURL` and the `commandExtension` or the `stateExtension` (if provided) the URL is formatted using the [java.util.Formatter](http://docs.oracle.com/javase/6/docs/api/java/util/Formatter.html).
+The URL is used as format string and two parameters are added:
+
+- the current date (referenced as `%1$`)
+- the transformed command (referenced as `%2$`)
+
+After the parameter reference the format needs to be appended.
+See the link above for more information about the available format parameters (e.g. to use the string representation, you need to append `s` to the reference).
+When sending an OFF command on 2020-07-06, the URL
+
+```
+http://www.domain.org/home/lights/23871/?status=%2$s&date=%1$tY-%1$tm-%1$td
+``` 
+
+is transformed to 
+
+```
+http://www.domain.org/home/lights/23871/?status=OFF&date=2020-07-06
+```
+
+## Full Example
+
+_Provide a full usage example based on textual configuration files (*.things, *.items, *.sitemap)._
```

Comment:
Looks like examples still need to be added in.

Reply:
Good question, is this still needed in OH3? We won't merge before.